Code/RDGeneral/ConcurrentQueue.h

```diff
@@ -0,0 +1,102 @@
+#ifdef RDK_THREADSAFE_SSS
+#ifndef CONCURRENT_QUEUE
+#define CONCURRENT_QUEUE
+#include <condition_variable>
+#include <queue>
+#include <thread>
+
+namespace RDKit {
+template <class E>
+class ConcurrentQueue {
```

Comment:
I believe that we do not want an instance of a `ConcurrentQueue` to be copyable. Please disable copying of this class.